bower.json

```diff
@@ -5,25 +5,22 @@
   "license": "MIT",
   "version": "1.2.0",
   "dependencies": {
-    "jquery": "~2.1.1",
-    "angular-animate": "~1.4.9",
-    "angular-cookies": "~1.4.9",
-    "angular-touch": "~1.4.9",
-    "angular-sanitize": "~1.4.9",
-    "angular-ui-router": "~0.2.17",
-    "angular-material": "1.0.0",
-    "angular": "1.4.9",
-    "angular-nvd3": "1.0.0-rc.2",
-    "components-font-awesome": "~4.5.0",
-    "angular-resource": "~1.5.0",
-    "angular-material-data-table": "^0.9.12",
+    "jquery": "^3.2.1",
+    "angular": "^1.6.5",
+    "angular-material": "^1.1.4",
```

Comment:
@botraunak After upgrading the versions, did you tested the application once entirely so we don't have any issues arising due to change in version ?